.gitignore

```diff
@@ -1,2 +1,10 @@
+# Generated by Cargo
+# will have compiled files and executables
+**/target/
 /target
-/Cargo.lock
+
+# These are backup files generated by rustfmt
+**/*.rs.bk
+
+# Unneeded artifacts
+**/.DS_Store
```

Comment:
only relevant on macos, belongs in the local .gitignore

Reply:
Removed!